src/PhpHash.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace Aws;
 
+use HashContext;
```

Comment:
I don't believe doc blocks will require having the `use` statement for the class.

Reply:
That's an acceptable change, it's done [in other places](https://github.com/aws/aws-sdk-php/blob/88171efe2ad8b048cd0459957ae69eaf656dbf80/src/S3/StreamWrapper.php#L79).